src/main/java/com/mbcoder/iot/weatherstation/Controller.java

```diff
@@ -55,7 +55,7 @@ public class Controller implements Initializable {
   @FXML private Gauge barometerGauge;
   private Timer recordingTimer; // timer for reading from the sensors
   private Timer loggingTimer; // timer for logging data to the feature service
-  private final boolean simulatedMode = false;
+  private final boolean simulatedMode = true;
```

Comment:
a side comment here might be useful so anyone using this repo plugged into a RPi knows to toggle this back off